net-print/cnijfilter-drivers/cnijfilter-drivers-3.80.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+MY_PN="${PN/-drivers/}"
+
+inherit eutils autotools flag-o-matic multilib
+
+DESCRIPTION="Canon InkJet Printer Driver for Linux (Pixus/Pixma-Series)."
+HOMEPAGE="http://support-sg.canon-asia.com/contents/SG/EN/0100469302.html"
+SRC_URI="http://gdlp01.c-wss.com/gds/3/0100004693/01/${MY_PN}-source-${PV}-1.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="${PV}"
+KEYWORDS="~amd64 ~x86"
+PRINTER_USE=( mp230 mg2200 e510 mg3200 mg4200 ip7200 mg5400 mg6300 )
+PRINTER_ID=( 401 402 403 404 405 406 407 408 )
+IUSE="${PRINTER_USE[@]} +servicetools"
+
+RDEPEND="
+	net-print/cnijfilter[servicetools?]
+	>=media-libs/libpng-1.5
+	>=media-libs/tiff-3.4
+	>=net-print/cups-1.4
+	servicetools? (
+		>=dev-libs/libxml2-2.7.3-r2
+		>=x11-libs/gtk+-2.6:2
+	)
+"
+DEPEND="${DEPEND}
+	sys-devel/gettext
+"
+
+REQUIRED_USE="|| ( ${PRINTER_USE[@]} )"
+
+S="${WORKDIR}/${MY_PN}-source-${PV}-1"
+
+_dir_build() {
```

Comment:
This looks like a lot of indirection for little value. Kill it.

Reply:
Fixed

Reply:
Fixed